certz/certz.proto

```diff
@@ -185,6 +220,30 @@ service Certz {
   rpc Rotate(stream RotateCertificateRequest)
       returns (stream RotateCertificateResponse);
 
+  // AddSSLProfile is part of SSL profile management and allows for adding new
```

Comment:
Maybe just `AddProfile`? Not sure whether there are other things that we'd `Add` within this service? (Just wondering whether we can make the naming shorter :-))

Reply:
OK. Done